src/coreclr/jit/emitarm64.cpp

```diff
@@ -2539,6 +2539,27 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(isVectorRegister(id->idReg2()));
             break;
 
+        case IF_SVE_BH_3A: // .........x.mmmmm ....hhnnnnnddddd -- SVE address generation
+            imm      = emitGetInsSC(id);
+            elemsize = id->idOpSize();
```

Comment:
Did you mean to check `elemsize` is `EA_SCALABLE` here?

Reply:
Yea I think I did mean to do that.